src/emulator/databaseEmulator.ts

```diff
@@ -173,12 +181,41 @@ export class DatabaseEmulator implements EmulatorInstance {
       if (e.context && e.context.body) {
         throw e.context.body.error;
       }
-      throw e.original;
+      if (e.original) {
+        throw e.original;
```

Comment:
Nit: Does` throw e.original ?? e` work as I hope it does? If so, i think its slightly more elegant.

Reply:
https://github.com/firebase/firebase-tools/pull/4490